website/docs/d/nginx_deployment.html.markdown

```diff
@@ -27,84 +27,84 @@ output "id" {
 
 The following arguments are supported:
 
-* `name` - (Required) The name of this Nginx Deployment.
+* `name` - (Required) The name of this NGINX Deployment.
 
-* `resource_group_name` - (Required) The name of the Resource Group where the Nginx Deployment exists.
+* `resource_group_name` - (Required) The name of the Resource Group where the NGINX Deployment exists.
 
 ## Attributes Reference
 
 In addition to the Arguments listed above - the following Attributes are exported:
 
-* `id` - The ID of the Nginx Deployment.
+* `id` - The ID of the NGINX Deployment.
 
-* `capacity` - The number of NGINX capacity units for this Nginx Deployment.
+* `capacity` - The number of NGINX capacity units for this NGINX Deployment.
 
 * `diagnose_support_enabled` - Whether diagnostic settings are enabled.
 
-* `email` - Preferred email associated with the Nginx Deployment.
+* `email` - Preferred email associated with the NGINX Deployment.
 
 * `frontend_private` - A `frontend_private` block as defined below.
 
 * `frontend_public` - A `frontend_public` block as defined below.
 
 * `identity` - A `identity` block as defined below.
 
-* `ip_address` - The IP address of the Nginx Deployment.
+* `ip_address` - The IP address of the NGINX Deployment.
 
-* `location` - The Azure Region where the Nginx Deployment exists.
+* `location` - The Azure Region where the NGINX Deployment exists.
 
 * `logging_storage_account` - A `logging_storage_account` block as defined below.
 
-* `managed_resource_group` - Auto-generated managed resource group for the Nginx Deployment.
+* `managed_resource_group` - Auto-generated managed resource group for the NGINX Deployment.
 
 * `network_interface` - A `network_interface` block as defined below.
 
-* `nginx_version` - NGINX version of the Nginx Deployment.
+* `nginx_version` - NGINX version of the Deployment.
 
-* `sku` - Name of the SKU for this Nginx Deployment.
+* `sku` - Name of the SKU for this NGINX Deployment.
 
-* `tags` - A mapping of tags assigned to the Nginx Deployment.
+* `tags` - A mapping of tags assigned to the NGINX Deployment.
 
 ---
 
 A `frontend_private` block exports the following:
 
-* `allocation_method` - The method of allocating the private IP to the Nginx Deployment.
+* `allocation_method` - The method of allocating the private IP to the NGINX Deployment.
 
-* `ip_address` - Private IP address of the Nginx Deployment.
+* `ip_address` - Private IP address of the NGINX Deployment.
 
-* `subnet_id` - The subnet ID of the Nginx Deployment.
+* `subnet_id` - The subnet ID of the NGINX Deployment.
 
 ---
 
 A `frontend_public` block exports the following:
 
-* `ip_address` - List of public IPs of the Ngix Deployment.
+* `ip_address` - Specifies a list of Public IP Resource IDs for this NGINX Deployment.
```

Comment:
this is a data source right? You should make it sound like a READ property. Right now it sounds like a user can specify this (which is what the resource docs should say).

Reply:
Agreed, this should be 
```suggestion
* `ip_address` - The list of Public IP Resource IDs for this NGINX Deployment.
```